python/ray/tests/test_logging.py

```diff
@@ -143,6 +144,43 @@ def is_event_loop_stats_found(path):
             wait_for_condition(lambda: is_event_loop_stats_found(path))
 
 
+def test_worker_id_names(shutdown_only):
+    ray.init(
+        num_cpus=1,
+        _system_config={
+            "event_stats_print_interval_ms": 100,
+            "event_stats": True
+        })
+    session_dir = ray.worker.global_worker.node.address_info["session_dir"]
+    session_path = Path(session_dir)
+    log_dir_path = session_path / "logs"
+
+    # Run the basic workload.
+    @ray.remote
+    def f():
+        print("hello")
+
+    ray.get(f.remote())
+
+    paths = list(log_dir_path.iterdir())
+
+    ids = []
+    for path in paths:
+        if "python-core-worker" in str(path):
+            pattern = ".*-([a-f0-9]*).*"
+        elif "worker" in str(path):
+            pattern = ".*worker-([a-f0-9]*)-.*-.*"
+        else:
+            continue
+        worker_id = re.match(pattern, str(path)).group(1)
+        ids.append(worker_id)
+    counts = Counter(ids).values()
+    for count in counts:
+        # There should be a "python-core-.*.log", "worker-.*.out",
+        # and "worker-.*.err"
+        assert count == 3
+
```

Comment:
Could you do an assert equals here? i.e.  something like: 
```
counts == {"python-core-.*." : 3}
```

Reply:
hmmm the key is just the worker id (rest of file name is parsed away). Since the files have different structures (`worker-.*` vs `python-core-worker-.*`), what would this the proposed change check that's different from the current one?

Reply:
Ahh okay! This does make sense :)